src/runtime_src/core/edge/user/zynq_dev.cpp

```diff
@@ -164,15 +164,19 @@ get_render_devname()
     // A symlink to render device is created based on this node name
     static const std::regex filter{"platform.*zyxclmm_drm-render"};
 
-    boost::filesystem::directory_iterator end_itr;
-    for( boost::filesystem::directory_iterator itr( render_dev_sym_dir ); itr != end_itr; ++itr) {
-        if (!std::regex_match(itr->path().filename().string(), filter))
-	    continue;
+    try {
+        boost::filesystem::directory_iterator end_itr;
+        for( boost::filesystem::directory_iterator itr( render_dev_sym_dir ); itr != end_itr; ++itr) {
```

Comment:
While you are hot cleaning up the mess :-)
```suggestion
        for (boost::filesystem::directory_iterator itr { render_dev_sym_dir }; itr != end_itr; ++itr) {
```

Reply:
Sure I can do that.

I am attempting to focus requests on the exact changes. They will get more complicated we progress with the merges.